### PR TITLE
#37 Return input args in promise chaining

### PR DESCRIPTION
**Pull request by fjcalzado**

### CODE REVIEW

File: src/trackPromise.js [hunk 1]
```diff
@@ -15,10 +15,11 @@ export const trackPromise = (promise, area) => {
   const promiseInProgress = anyPromiseInProgress(area);
   emitter.emit(promiseCounterUpdateEventId, promiseInProgress, area);
 
-  promise
-    .then(() => decrementPromiseCounter(area),
-      () => decrementPromiseCounter(area)
-    );
+  const onResolveHandler = (arg) => {
```
Comment (nasdan) [inline src/trackPromise.js]: Does it resolve the issue?

Why is it working on 1.0.2 version and not in 1.1.0? Seeing last PR the code is the same except the area parameter

Comment (brauliodiez) [reply src/trackPromise.js]: Is related with another issue

**Comment by brauliodiez** [comment]
It would be great to add a unit test to check that the case is solved and as documentatin

File: src/trackPromise.js (hunk 1 above)
Comment (fjcalzado) [reply src/trackPromise.js]: No it doesn't. The issue in 1.1.0 is not related with this enhancement, but while investigating it I came across with this improvement. I will soon release a fix for the issue. We are discussing different approaches.

**Comment by fjcalzado** [comment]
After implementing the unit test, the fix is not needed as we are returning the same input promise. However, it its interesting to keep the unit test for future proof. So, this PR should be merged to add the unit test.

**Pull request merged**